Framework/script/o2-qc-multinode-test.sh

```diff
@@ -42,29 +49,20 @@ function delete_data() {
   curl -i -L ccdb-test.cern.ch:8080/truncate/qc/TST/QO/MultiNodeLocalTest
   curl -i -L ccdb-test.cern.ch:8080/truncate/qc/TST/QO/MultiNodeRemoteTest
 
-  rm -f /tmp/multinode_test_obj${UNIQUE_PORT_1}.root
-  rm -f /tmp/multinode_test_obj${UNIQUE_PORT_2}.root
+  cd /tmp
+  # mv in /tmp is guaranteed to be atomic
+  mv /tmp/${UNIQUE_TEST_NAME}{,.todelete}
+  rm -rf /tmp/${UNIQUE_TEST_NAME}.todelete
 }
 
-if [ -z "$UNIQUE_PORT_1" ]
-then
-  echo "UNIQUE_PORT_1 must be set when calling o2-qc-multinode-test.sh"
-  exit 1
-fi
+# mkdir in /tmp is guaranteed to be atomic
+mkdir /tmp/${UNIQUE_TEST_NAME} || { echo "Concurrent usage of the same port ${UNIQUE_PORT_1} detected, exiting"; exit 1; }
+pushd /tmp/${UNIQUE_TEST_NAME}
+
+UNIQUE_PORT_2=$((UNIQUE_PORT_1+1))
```

Comment:
To avoid conflicts, the cmake code should be changed to have only odd / even ports.